app/Models/Notification/Notification.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Models\Notification;
+
+use App\Models\Controller\ControllerPosition;
+use App\Models\User\User;
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+class Notification extends Model
+{
+    protected $fillable = [
+        'title',
+        'body',
+        'valid_from',
+        'valid_to',
+        'disabled_at'
+    ];
+
+    public function scopeActive($query)
+    {
+        return $query->where('valid_from', '<=', Carbon::now())
```

Comment:
If you decide to go with `SoftDeletes` you can remove the disabled_at check as that's handled automatically by the trait

Reply:
68a5d3c